Lets Do This/Controllers/Campaign/LDTCampaignDetailViewController.m

```diff
@@ -91,6 +91,19 @@ - (void)viewDidLayoutSubviews {
     self.collectionView.contentInset = UIEdgeInsetsMake(0,0,0,0);
 }
 
+// Attempts fix for the item width must be less than the width of the UICollectionView minus the section insets left and right values, minus the content insets left and right values
+//
+//- (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator {
+//    [super viewWillTransitionToSize:size withTransitionCoordinator:coordinator];
+//
+//    [self.collectionView.collectionViewLayout invalidateLayout];
+//}
+
+- (void)viewWillLayoutSubviews {
+    [super viewWillLayoutSubviews];
+    self.flowLayout.estimatedItemSize = CGSizeMake(CGRectGetWidth(self.collectionView.bounds), 400);
```

Comment:
Setting this in `viewDidLoad` like the tutorial suggests causes xcode to go cray and the view never loads.